src/app/docs-pages/keyboard-shortcuts-doc/keyboard-shortcuts-doc.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="doc-page-container">
+
+  <h1 class="doc-title">Keyboard Shortcuts Service</h1>
+
+  <p class="doc-info">
+    The service lets you enable keyboard-shortcuts all over in the application. 
+    Enabling the service in your app-component makes the shortcuts available everywhere. 
+    You therefore only have to enable the service once!
+  </p>
+  <p class="doc-info">
+    To open the overview of shortcuts press ctrl and space. 
+    This overview lists all the shortcuts specifically for that component.
```

Comment:
Endre til noe ala: "This an automaticly generated overview of all active key compos on the current page"